src/stories/cartesian/brush/BrushComponent.jsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable no-useless-constructor */
+/* eslint-disable import/no-default-export */
+import React, { Component } from 'react';
+import { Surface, Brush } from '../../..';
+
+export default class BrushComponent extends Component {
```

Comment:
I think it is preferable to only have a single file per component. (Brush, in this case). 
See https://github.com/recharts/recharts/blob/34ccf85ec4681f95813d78ed52dad1e79e51f695/src/stories/LineChart.stories.tsx for example.

Reply:
Really, this is not the BrushComponent, but a story template and should instead live here 
```
function Template(args) {
  return <BrushComponent {...args} />;
}
```

Reply:
I was thinking of just integrating what dillonreedy had been working on.
I think that's better than starting with a storybook from nothing.

Reply:
Well each has pros and cons. Integrating @dillonreedy s work gives us a nice set a baseline stories, but we will have to refactor it. 

I think we should integrate his work indeed, but only after agreeing on a best practice setup with a single story. We need to be aligned on the baseline / expectation and then we can write issues to integrate in the same manner.